requirements/testing.txt

```diff
@@ -111,6 +111,8 @@ pydata-google-auth==1.2.0
     # via pandas-gbq
 pyfakefs==4.5.6
     # via -r requirements/testing.in
+pyhive[presto]==0.6.5
+    # via apache-superset
```

Comment:
did this come in from somewhere else?

Reply:
When using master I got this, using flask-caching==1.11.0 I get no changes here. 🤔